bot/exts/fun/hangman.py

```diff
@@ -0,0 +1,195 @@
+        ))
+
+        # Game loop
+        while user_guess.replace(' ', '') != word:
+            # Start of the game
+            await original_message.edit(embed=self.create_embed(tries, user_guess))
+
+            # Sends a message if they do not guess within 60 seconds
+            try:
+                message = await self.bot.wait_for(
+                    event="message",
+                    timeout=60.0,
+                    check=check
+                )
+            except TimeoutError:
+                timeout_embed = Embed(
+                    title=choice(NEGATIVE_REPLIES),
+                    description="Looks like the bot timed out! You must send a letter within 60 seconds.",
+                    color=Colours.soft_red,
+                )
+                await original_message.edit(embed=timeout_embed)
+                return
+
+            # If the user enters a capital letter as their guess, it is automatically converted to a lowercase letter
+            normalized_content = message.content.lower()
+            # Alerts the user if they enter more than one letter per guess
+            if len(normalized_content) > 1:
+                letter_embed = Embed(
+                    title=choice(NEGATIVE_REPLIES),
+                    description="You can only send one letter at a time, try again!",
+                    color=Colours.dark_green,
+                )
+                await ctx.send(embed=letter_embed, delete_after=4)
+                continue
+
+            # Checks for repeated guesses
+            elif normalized_content in guessed_letters:
+                already_guessed_embed = Embed(
+                    title=choice(NEGATIVE_REPLIES),
+                    description=f"You have already guessed `{normalized_content}`, try again!",
+                    color=Colours.dark_green,
+                )
+                await ctx.send(embed=already_guessed_embed, delete_after=4)
+                continue
+
+            # Checks for correct guesses from the user
+            elif normalized_content in word:
+                positions = {idx for idx, letter in enumerate(pretty_word) if letter == normalized_content}
+                user_guess = "".join(
+                    [normalized_content if index in positions else dash for index, dash in enumerate(user_guess)]
+                )
+
+            else:
+                tries -= 1
+
+                if tries <= 0:
+                    losing_embed = Embed(
+                        title="You lost.",
+                        description=f"The word was `{word}`.",
+                        color=Colours.soft_red,
+                    )
+                    await original_message.edit(embed=self.create_embed(tries, user_guess))
+                    await ctx.send(embed=losing_embed)
+                    return
+
+            guessed_letters.add(normalized_content)
+
+        # Send the message saying that you won and update the game board
```

Comment:
This doesn't really tell us more than we understand from the code right below it. If you do want to keep a comment here let's make it about something that while not impossible to know without it, may not be awfully apparent.
```suggestion
        # The loop exited meaning that the user has guessed the word
```